test/TPM_Didymos.jl

```diff
@@ -77,18 +77,18 @@
 
     ##= TPM =##
     thermo_params = AsteroidThermoPhysicalModels.ThermoParams(  # [Michel+2016; Naidu+2020]
-        A_B   = 0.059,  # Bolometric Bond albedo
-        A_TH  = 0.0,
-        k     = 0.125,
-        ρ     = 2170.,
-        Cp    = 600.,
-        ε     = 0.9,
-        t_bgn = et_range[begin],
-        t_end = et_range[end],
-        Nt    = length(et_range),
-        z_max = 0.6,
-        Nz    = 41,
-        P     = SPICE.convrt(AsteroidThermoPhysicalModels.DIDYMOS[:P], "hours", "seconds"),
+        A_B     = 0.059,  # Bolometric Bond albedo
+        A_TH    = 0.0,
+        k       = 0.125,
+        ρ       = 2170.,
+        Cp      = 600.,
+        ε       = 0.9,
+        t_begin = et_range[begin],
+        t_end   = et_range[end],
+        Nt      = length(et_range),
+        z_max   = 0.6,
+        Nz      = 41,
+        P       = SPICE.convrt(AsteroidThermoPhysicalModels.DIDYMOS[:P], "hours", "seconds"),
     );
```

Comment:
```suggestion
    )
```
Could you also remove this unnecessary semicolon?